torch/_inductor/decomposition.py

```diff
@@ -26,7 +26,9 @@
         aten.bitwise_and_,
         aten.bitwise_or_,
         aten.clamp_min_,
+        aten.empty_like,
         aten.flip,
+        aten.full_like,
```

Comment:
The lines here say which decomps in torch._decomps and torch._refs to use, no need to add things here if you define them below.